### PR TITLE
Install OpenBLAS for mxnet1.8.0

### PR DESCRIPTION
**Pull request by crcrpar**
Install OpenBLAS for mxnet1.8.0 to successfully dlopen libopenblas.so.0.

### CODE REVIEW

**Comment by hvy** [comment]
Thanks for the quick PR. In case it works, I'd suggest we leave comments.
By the way, https://pypi.org/project/mxnet/ states that `libquadmath0` is required. Is this a different topic?

**Comment by hvy** [comment]
Could you also update the [coverage.yml](https://github.com/optuna/optuna/blob/master/.github/workflows/coverage.yml) ?

**Comment by crcrpar** [comment]
Updated examples.yml as well as coverage.yml.

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Install (apt)
       run: |
         sudo apt-get update
-        sudo apt-get -y install openmpi-bin libopenmpi-dev
+        sudo apt-get -y install openmpi-bin libopenmpi-dev h
```
Comment (hvy) [inline .github/workflows/examples.yml:36]: Is this `h` intentional?
```suggestion
        sudo apt-get -y install openmpi-bin libopenmpi-dev
```

Comment (crcrpar) [reply .github/workflows/examples.yml]: not, I don't know where this h comes from and what this h means.

**Review by hvy — APPROVED** [review]
LGTM!

**Review by HideakiImamura — COMMENTED** [review]
`examples` includes MXnet exampels, but `.github/workflows/examples.yml` is not updated. Is it intentional?

File: Dockerfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ FROM python:${PYTHON_VERSION}
 ENV PIP_OPTIONS "--no-cache-dir --progress-bar off"
 
 RUN apt-get update \
-    && apt-get -y install openmpi-bin libopenmpi-dev \
+    && apt-get -y install openmpi-bin libopenmpi-dev libopenblas-dev\
```
Comment (toshihikoyanase) [inline Dockerfile:8]: How about adding a whitespace before `\` like the other lines?
```suggestion
    && apt-get -y install openmpi-bin libopenmpi-dev libopenblas-dev \
```

**Review by HideakiImamura — APPROVED** [review]
Thanks for your swift actions! LGTM!

**Pull request merged**